data/plot.py

```diff
@@ -31,22 +31,22 @@ def center_x(price):
         middle_price = (max_price - min_price) / 2 + min_price
 
         price = "%d" % max_price
-        draw.text((center_x(price), position_first[1]), price, font=font)
+        draw.text((center_x(price), position_first[1]), price, font=font, fill=fill)
         price = "%d" % middle_price
-        draw.text((center_x(price), (position_last[1] - position_first[1]) / 2 + position_first[1]), price, font=font)
+        draw.text((center_x(price), (position_last[1] - position_first[1]) / 2 + position_first[1]), price, font=font, fill=fill)
         price = "%d" % min_price
-        draw.text((center_x(price), position_last[1]), price, font=font)
+        draw.text((center_x(price), position_last[1]), price, font=font, fill=fill)
 
     @staticmethod
-    def caption(price, y, screen_width, font, draw):
-        draw.text((-1, y), "BTC", font=font)
+    def caption(price, x, y, screen_width, font, draw, fill=None):
```

Comment:
As the method signature was changed and no default parameter provided the other screens will fail.